irina/ex.js

```diff
@@ -0,0 +1,48 @@
+// # Programming Basics: Ternary
+
+// These exercises are designed to make you more comfortable with the ternary operator. 
+
+// 1. Declare a variable named isDog. If true, print 'pat, pat' and if not, print 'find me a dog to pat!'
+let isDog
```

Comment:
1. Declare a variable named isDog. If true, print 'pat, pat' and if not, print 'find me a dog to pat!

in this case isDog is not boolean